wiki/Client/Release_stream/Lazer/Feature_comparison/en.md

```diff
@@ -0,0 +1,438 @@
+  - Automatic beatmap queues
+- New editor features
+  - Per-segment curve types for sliders
+  - Slider splitting and merging
+  - New "verify" tab
+- Support for custom game modes
+- Gameplay
+  - Two new built-in skins
+  - Replay seeking
+  - Beatmap (local) offset auto-calibration
+- Built-in skin editor with customisable skinning components 
+  - Song density graph
+  - Performance points counter
+  - Unstable rate counter
+  - Longest combo counter
+  - Custom text components, beatmap information displays
+  - Custom static sprites
+  - Judgement counter
+  - Clicks per second counter
+- Snaking out sliders in the osu! mode
+- No black bar behind the playfield in osu!taiko
+- Timing-based note colouring in osu!mania
+
+<!-- - osu!catch ?-->
+
+## Gameplay differences 
+
+<!--
+- Multiplayer
+  - TAG co-op mode doesn't exist
+-->
+
+### General
+
+#### Song playback ramps up and down when pausing and unpausing
+
+This makes it harder to continue from a pause in the middle of active gameplay, preventing abuse of pausing to some extent.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | No |
+| Classic mod revert support | No |
+| Intentionally changed | Yes |
+| Needs further consideration | Yes |
+
+#### The Flashlight gradient does not match stable
+
+<!-- comparison images for relevant game modes  -->
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | No |
+| Classic mod revert support | No |
+| Intentionally changed | No |
+| Needs further consideration | Yes |
+
+#### The Easy mod no longer pauses gameplay when recovering from failure
+
+<!-- paragraph? -->
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | No |
+| Classic mod revert support | No |
+| Intentionally changed | No |
+| Needs further consideration | No? |
+
+#### The health system is different
+
+This is mostly due to changes to the judgement system. For example, Geki and Katu judgements do not exist in lazer, which affected health on stable.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | No? |
+| Needs further consideration | Yes |
+
+#### All game modes use the same grading system
+
+All game modes use a grading system similar to the one in osu!mania on stable (c.f. [Grade](/wiki/Gameplay/Grade)).
+
+| Grade | Condition |
+| :-: | :-- |
+| SS | 100% accuracy |
+| S | At least 95% accuracy |
+| A | At least 90% accuracy |
+| B | At least 80% accuracy |
+| C | At least 70% accuracy |
+| D | Anything else |
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | Yes |
+| Needs further consideration | Yes? |
+
+#### Hitsound volume cannot be adjusted in between notes
+
+In stable, it is possible to change the volume of ghost taps in osu!taiko and osu!mania, or create gradual osu!taiko swell volume changes. In lazer, there is only per-note volume, making that impossible.
```

Comment:
This seems like a bug, and I'm not sure it deserves to be listed here. Probably better off making an issue if one doesn't exist (I have a strong feeling one does already though).

Reply:
indeed: https://github.com/ppy/osu/discussions/17247. i got most of these issues from either open issues, discussions or while researching stable code. i do need to add links later as mentioned in op

> In stable, you can change the volume of "ghost taps", or create fluent swell volume changes. In Lazer, there's only per-note volume, making that impossible.

perhaps it should be split off to a separate issue

Reply:
I've tested and indeed it doesn't seem to work. I'll take a look into this today and open an issue if it's not a simple fix.

Reply:
https://github.com/ppy/osu/pull/22659 :)

Reply:
i've opted for a comment that links to the above pr for now

Reply:
It's merged so please remove this section now.